client/packages/system/src/Vaccination/Components/VaccineCardTable.tsx

```diff
@@ -124,23 +124,40 @@ export const VaccinationCardComponent: FC<VaccinationCardProps> = ({
 
   useStyleRowsByStatus(data?.items, isEncounter);
 
+  const getAgeLabel = (row: VaccinationCardItemFragment) => {
```

Comment:
this is the juicy bit frontend wise

Reply:
Yep - discussion with design team was that the grouping by age is most important, and from there it makes more sense to all be under one label than awkwardly have some lines of each kind of label. Added to the docs that the expectation is that if you are using custom labels, the same label should be used for all doses of that age, across all courses 🙏  (Ideally we'd have the labels as some kind of reusable chip select thing, but that was too much to implement at this stage of the release!)